app/server/utils/user.py

```diff
@@ -511,6 +511,15 @@ def proccess_create_or_modify_user_request(
         }
         return response_object, 400
 
+    business_usage = None
+    if business_usage_id:
```

Comment:
`business_usage_id` was changed to `business_usage` on line 175. Looks like an error

Also `business_usage = None` is unnecessary as default None in `create_transfer_account_user` params.

Reply:
@enjeyw updated comment to make it clearer what I was saying ehre

Reply:
line 175 is a different function... this function has `business_usage_id` defined higher up in it?*
 and the None is required because otherwise u'd be passing in an undefined param into `create_transfer_account_user`

*That being said the code above where it's defined is kinda weird... i think business usage creation  needs a review at some point.

Also I have exactly ZERO recollection of writing these lines lol